packages/react-reconciler/src/ReactFiberRoot.js

```diff
@@ -91,6 +100,7 @@ type ProfilingOnlyFiberRootProperties = {|
 export type FiberRoot = {
```

Comment:
While we're here, should this have been exact?

Reply:
I don't think Flow supported the spreading as exact but arguably we shouldn't do the spreading neither. We don't for Fiber.